src/types.js

```diff
@@ -489,18 +489,6 @@ export type GetState = () => GlobalState;
 
 export type LocalizableText = any; // string | { text: string, values: Object };
 
-export type RenderedTimeDescriptor = {
```

Comment:
I am not sure you need to delete these items in a separate commit (even though they are indeed not used)
In the currently last commit, you are reintroducing them under slightly different names.

Just use the last commit to replace these and skip this commit altogether.